### PR TITLE
Et/mapf

### PR DESCRIPTION
**Pull request by EtienneTignon**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -20644,6 +20644,12 @@ @inproceedings{salste20a
   pages = {1711-1715}
 }
 
+@inproceedings{salube12,
```
Comment (tortinator) [inline krr.bib:20647]:
```suggestion
@inproceedings{salube12a,
```

File: procs.bib [hunk 2]
```diff
@@ -3771,6 +3771,14 @@ @proceedings{socs11
   year = {2011}
 }
 
+@proceedings{socs12,
+  title = {Proceedings of the Fourth Annual Symposium on Combinatorial Search (SOCS'12)},
+  editor = {D. Borrajo and A. Felner and R. Korf and M. Likhachev and C. Linares L{\'{o}}pez and W. Ruml and N. Sturtevant},
```
Comment (tortinator) [inline procs.bib:3776]:
```suggestion
  editor = {D. Borrajo and A. Felner and R. Korf and M. Likhachev and C. {Linares L{\'{o}}pez} and W. Ruml and N. Sturtevant},
```

**Review by tortinator — CHANGES_REQUESTED** [review]

**Comment by EtienneTignon** [comment]
I committed the requested changes and checked the format locally afterward.

**Comment by jorgefandinno** [comment]
I get the following error
```
You can't pop an empty literal stack for entry gekakaosscwa16a
while executing---line 1338 of file mdpi.bst
You can't pop an empty literal stack for entry liesus16a
while executing---line 1338 of file mdpi.bst
```

It also seems that bibliography commit is not up to date. May be this the reason for the errors?

**Comment by tortinator** [comment]
Have you updated the submodule..?

**Comment by jorgefandinno** [comment]
I think so. I do have a folder ```include/latex-class-mdpi``` with a bunch of new files.

Is that enough to be sure?

**Comment by tortinator** [comment]
Sorry but have you cleaned all six files, in particular, the ones related to bibtex...?

**Comment by jorgefandinno** [comment]
> Sorry but have you cleaned all six files, in particular, the ones related to bibtex...?

Yes, I even cloned the repository in a new location and get the same error.

**Review by tortinator — APPROVED** [review]

**Comment by tortinator** [comment]
@jorgefandinno I don't know - nonetheless: What dooes this have to do with @EtienneTignon's pull request..?

**Comment by tortinator** [comment]
> > Sorry but have you cleaned all six files, in particular, the ones related to bibtex...?
> 
> Yes, I even cloned the repository in a new location and get the same error.

I did the same but obtain no error ....

**Pull request merged**